tests/epw_test.py

```diff
@@ -18,11 +19,34 @@ def tearDown(self):
 
     def test_import_epw(self):
         """Test import standard epw."""
-        pass
-
+        relative_path = 'tests\\epw\\chicago.epw'
+        abs_path = os.path.abspath(relative_path)
+
+        epw_rel = EPW(relative_path)
+        epw = EPW(abs_path)
+
+        assert epw_rel.file_path == relative_path
+        assert epw_rel.location.city == 'Chicago Ohare Intl Ap'
+        assert epw.file_path == abs_path
+        assert epw.location.city == 'Chicago Ohare Intl Ap'
+        # Check that calling location getter only retrieves location
+        assert epw.is_data_loaded == False
+        epw.import_data()
+        assert epw.is_data_loaded == True
+
+    # Should these tests fail? I'm not sure I understand what the Tokyo.epw is
```

Comment:
It's a non-standard epw file generated by one of the users. It used to fail but doesn't fail anymore! It's part of the test to ensure we can handle non-standard epw weather files.